webpack.config.js

```diff
@@ -1,31 +1,45 @@
-let path = require('path');
+const path = require('path');
 
-module.exports = env => {
-    console.log('env: ', env);
+const config = {
+  entry: path.resolve(__dirname, 'src/client/index'),
+  output: {
+    path: `${__dirname}/dist`,
```

Comment:
Can be changed to path.resolve(__dirname, 'dist') for consistency, but also I'm not sure it's needed to specify full path.